bin/GameIndex.dbf

```diff
@@ -9730,7 +9730,7 @@ Compat = 4 // This game needs the EE cache to work which is only supported in "i
 	author=PSI
 	// The game allocates a VIF DMA buffer on the stack and sends it, returns from the function, then calls another function to wait for the transfer to end.
 	// The second function causes stack corruption as it allocates over the buffer, which is still transferring data.
-	// This works on real hardware due to the data cache - the transfer will remain safe in main memory, and the stack corruption only happens in cache.
+	// This works on real Hardware due to the data cache - the transfer will remain safe in main memory, and the stack corruption only happens in cache.
```

Comment:
"hardware" is not a name, it's a word in the middle of a sentence, it doesn't get capitalised.